packages/swap/src/routes/swap/index.ts

```diff
@@ -66,10 +78,15 @@ router.get(
 
     let state: State;
 
-    if (swap?.egressCompletedAt) {
+    if (swap?.egress?.broadcast?.succeededAt) {
       assert(swap.swapExecutedAt, 'swapExecutedAt should not be null');
       assert(swap.egress, 'egress should not be null');
+      assert(swap.egress.broadcast, 'broadcast should not be null');
       state = State.Complete;
+    } else if (swap?.egress?.broadcast) {
+      assert(swap.swapExecutedAt, 'swapExecutedAt should not be null');
+      assert(swap.egress, 'egress should not be null');
```

Comment:
this is redundant?

Reply:
I think I originally added these in to ensure data consistency. Not sure if they bring us much anymore

Reply:
i wasnt sure about why we have that, so i just copied it from the other state tbh. will remove them 🙂